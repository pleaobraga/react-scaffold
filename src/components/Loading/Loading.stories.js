import React from 'react'
import { storiesOf } from '@storybook/react'
import Loading from './Loading'

export default storiesOf('Components | Loading', module).add(
  'default',
  () => <Loading />,
  { info: { inline: true, header: false } }
)
