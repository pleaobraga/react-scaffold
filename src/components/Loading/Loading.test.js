import React from 'react'
import { shallow } from 'enzyme'
import Loading from './Card'

describe('Card', () => {
  const card = shallow(<Loading />)

  it('render proprely', () => {
    expect(card).toMatchSnapshot()
  })
})
