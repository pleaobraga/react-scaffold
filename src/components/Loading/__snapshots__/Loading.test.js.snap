// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading render proprely 1`] = `ShallowWrapper {}`;
